doc/content/guides/connecting-gateways/kerlinkwirnet/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Kerlink Wirnet Station"
+description: ""
+weight: 1
+---
+
+Kerlink Wirnet Station is a LoRaWAN gateway, whose technical specifications can be found in [the official documentation](https://www.kerlink.com/product/wirnet-station/). This page guides you to connect it to The Things Stack.
+
+## Prerequisites
+
+1. User account on The Things Stack with rights to create Gateways and API Keys.
+2. Kerlink Wirnet Station:
+  1. For configuration via provisioning script: Common Packet Forwarder installed and enabled and/or gateway running firmware version `3.0` or higher.
+  2. For manual configuration: Common Packet Forwarder installed and enabled
+
+> NOTE: Minimum CPF version tested is `v1.1.6`
+
+## Registration
+
+Create a gateway by following the instructions for the [Console]({{< ref "/guides/getting-started/console#create-gateway" >}}) or the [CLI]({{< ref "/guides/getting-started/cli#create-gateway" >}}). Choose a **Gateway ID** that is at least 6 characters in length. An **EUI** is not necessary.
+
+Create an API Key with Gateway Link rights for this gateway using the same instructions. Copy the key and save it for later use.
+
+## Configuration
+
+All further steps will assume the gateway is available at `192.168.4.155` and the stack address is `https://thethings.example.com`, please replace these by the addresses appropriate for your setup.
+
+### Provisioning script
+
+Please refer to https://github.com/TheThingsNetwork/kerlink-station-firmware#provisioning for detailed up-to-date documentation.
+
+1. Ensure your public key is authorized for `root` user on the gateway. If that's not the case (you need to enter `root` password to login into the gateway via SSH) - execute `ssh-copy-id root@192.168.4.155`.
+2. Execute `curl -sL https://raw.githubusercontent.com/TheThingsNetwork/kerlink-station-firmware/master/provision.sh | bash -s -- "192.168.4.155" "https://thethings.example.com"`
```

Comment:
I considered using a permlink here, but IMO using `master` directly is a better approach here from perspective of maintenance of this guide. In case the location/usage changes we should update this doc as well.